hydra.yml

```diff
@@ -32,4 +32,8 @@ contracts:
     abi: dao-go/artifacts/voice/voice.abi
     name: voice
     wasm: dao-go/artifacts/voice/voice.wasm
+  cosale:
+    abi: dao-go/artifacts/vesting/vesting.abi
+    name: cosale
+    wasm: dao-go/artifacts/vesting/vesting.wasm
```

Comment:
This file is missing from this PR. I think you have to force add it.

Reply:
Hmm I did force add it, but since wasm is in gitignore, weird things may happen

Reply:
Could you try with `git add --force dao-go/artifacts/vesting/vesting.wasm` ? otherwise we need to clean up the .gitignore

Reply:
Ok did it again

Reply:
I think one of them was force added, but not the other, now both are force added

Reply:
I think you forgot to push it - I can't see the vesting.wasm file in the PR